project/LinkerdBuild.scala

```diff
@@ -368,15 +373,16 @@ object LinkerdBuild extends Base {
          |  GC_LOG="/var/log/namerd"
          |fi
          |
+         |# Check Java version for use in GC_LOG_OPTION and DEFAULT_JVM_OPTIONS
```

Comment:
Big pieces of these three execScripts are identical.  can we pull out common bits to avoid repeating ourselves?